### DISCUSSION TITLE
Need a single extension for c#/asp.net

### DISCUSSION THREAD

**Discussion by gsmental** [microsoft/vscode-discussions#162] [Extension Development QnA] [answered]
No doubt there are many extensions,  but there is no good extension, many extensions are obsoleted or not updated since many years. actually we want to reduce dependency on Visual studio.

**Comment by Ibrahim-Fouad** [comment, marked as answer]
There is an official extension from Microsoft called C#, which is powered by OmniSharp.
you can find on marketplace or [here](https://marketplace.visualstudio.com/items?itemName=ms-dotnettools.csharp).
